_locales/nl.yml

```diff
@@ -0,0 +1,42 @@
+configuration:
+  version: 1.3
+  label: Dutch
+  prefix: /nl/
+  available: false
```

Comment:
We will need this value to be `true` so Dutch can be displayed in the language picker in the sidebar. :)